GRDB/Core/Database+SQLCipher.swift

```diff
@@ -0,0 +1,211 @@
+#if SQLITE_HAS_CODEC
+import SQLCipher
+
+extension Database {
+    
+    /// Granularitly of SQLCipher log outputs
+    /// Each log level is more verbose than the last
+    ///
+    /// See https://www.zetetic.net/sqlcipher/sqlcipher-api/#cipher_log_level
+    public enum CipherLogLevel: String {
+        case none
+        case error
+        case warn
+        case info
+        case debug
+        case trace
+    }
+    
+    /// - Returns: the SQLCipher version
+    ///
+    /// See https://www.zetetic.net/sqlcipher/sqlcipher-api/#cipher_version
+    public var cipherVersion: String? {
+        get throws { try String.fetchOne(self, sql: "PRAGMA cipher_version") }
+    }
```

Comment:
My apologies for not suggesting the correct replacement in my first review 😅 

The correct form is:

```suggestion
    public var cipherVersion: String {
        get throws { try String.fetchOne(self, sql: "PRAGMA cipher_version")! }
    }
```

Explanation: `PRAGMA cipher_version` never returns NULL. We even check for it in `validateSQLCipher()` which is called very early after opening the connection and before yielding it to the user.